modules/tensor_mechanics/doc/content/documentation/systems/UserObjects/CrystalPlasticityStateVarRateComponentVoce.md

```diff
@@ -0,0 +1,59 @@
+# Crystal Plasticity State Var Rate Component Voce
+
+!syntax description /UserObjects/CrystalPlasticityStateVarRateComponentVoce
+
+## Description
+
+This UserObjects, `CrystalPlasticityStateVarRateComponentVoce` is meant to be used within the user object base crystal plasticity framework as a Voce hardening rule.
+
+\begin{equation}
+\label{eq:self_hardening_rate}
+h^{\alpha} = h_0^{\alpha}\left(1-\frac{\tau^{\alpha}_{CRSS}-\tau_0^{\alpha}}{\tau_{Sat}^{\alpha}-\tau_0^{\alpha}} \right)^{m^{\alpha}}
+\end{equation}
+
+with
+
+\begin{equation}
+\label{eq:slip_system_hardening_rate}
+\dot{\tau}^{\alpha}_{CRSS} = \sum_{\beta=1}^{N} \lvert \dot{\gamma}^\beta \rvert h^\beta q^{\alpha \beta}
+\end{equation}
+
+
+where: $\tau^{\alpha}_{CRSS}$ is the current critical resolved shear stress, $\tau^{\alpha}_{0}$ is initial critical resolved shear stress, $\tau_{Sat}^{\alpha} $ is the saturation resolved shear stress, $q^{\alpha \beta}$ is the matrix containing self and latent hardening coefficients; $h_0^{\alpha}$ and $m^\alpha$ are hardening parameters.
+
+Like for the others crystal plasticity routines values of all the parameters can be specified by groups of slip systems.
+
+This userobjects assumes that slip systems are provided in a well defined order and are grouped by planes (see  modules/tensor_mechanics/test/tests/cp_user_object/input_slip_sys_bcc48.txt for BCC crystals and modules/tensor_mechanics/test/tests/cp_user_object/input_slip_sys.txt for FCC)
+
+Seven variables need to be specified and one value is required for each group:
+
+
+   -  "groups" in which groups of slip systems are listed
+
+   -  "h0_group_values" $h_0$ value for each slip system group
+   
+   -  "tau0_group_values" $\tau_0$ value for each slip system group
+
+   -  "tauSat_group_values" = $\tau_{Sat}$ value for each slip system group
+
+   -  "hardeningExponent_group_values" $m$ value for each slip system group
+
+   -  "selfHardening_group_values" $q^{\alpha\alpha}$
+
+   -  "coplanarHardening_group_values" $q^{\alpha\beta}$ for co-planar slip systems in the same group
```

Comment:
This list does not render correctly: http://mooseframework.inl.gov/docs/PRs/11308/site/documentation/systems/UserObjects/CrystalPlasticityStateVarRateComponentVoce.html#c69da0ed-3b16-4f50-b11f-19854863b059

Do not indent and do not leave empty lines between list items